collectors/osv/constants.py

```diff
@@ -1,4 +1,8 @@
-from osidb.helpers import get_env
+from osidb.helpers import get_env, get_env_date
 
 # Switch to turn the collector on/off
 OSV_COLLECTOR_ENABLED = get_env("OSV_COLLECTOR_ENABLED", default="True", is_bool=True)
+
+SNIPPET_CREATION_START_DATE = get_env_date(
+    "SNIPPET_CREATION_START", default="2024-07-01"
```

Comment:
We use the same name for the cveorg collector but why the different default date?
What happens if we set the env `SNIPPET_CREATION_START=2024-07-01`, will the cveorg collector fail to create the snippets?